modules/popup-stack/lib/PopupStack.ts

```diff
@@ -212,31 +226,37 @@ export const PopupStack = {
    * method when the event triggers.
    */
   add(item: PopupStackItem): void {
+    const stack = getTopStack();
     if (stack._adapter?.add) {
       stack._adapter.add(item);
       return;
     }
     stack.items.push(item);
-    document.body.appendChild(item.element);
+    (stack.container?.() || document.body).appendChild(item.element);
```

Comment:
All the `|| document.body` is backwards compatible code. If this version of `PopupStack` isn't the first loaded, the `container` would not be defined. The `PopupStack` tries to register a global shared `window.workday.__popupStack` variable. This code cannot be removed.